### ISSUE TITLE
Sigil package segmentation fault on Ubuntu 16.04

### ISSUE THREAD

**Issue by michiels** [intercity/intercity-next#157]
I'd like to create this issue for archive/support on short term. It turns out that installing Dokku on Ubuntu 16.04 installs the wrong "sigil" package. Sigil is the template parser Dokku uses to for example parse nginx configuration files.

Because of the wrong install/package name/dokku dependency, the deploy of a Rails app will crash:

```
-----> Attempting to run scripts.dokku.postdeploy from app.json (if defined)
-----> Found previous container(s) (4e9cc24aa976) named blogtest.web.1
=====> renaming container (4e9cc24aa976) blogtest.web.1 to blogtest.web.1.1467711447
=====> renaming container (a51aee35107c) thirsty_brown to blogtest.web.1
-----> Configuring blogtest.firmhouse.com...(using built-in template)
remote: /var/lib/dokku/plugins/available/nginx-vhosts/functions: line 123: 29306 Segmentation fault      (core dumped) sigil "${SIGIL_PARAMS[@]}" > "$NGINX_CONF"
To dokku@198.211.127.154:blogtest
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'dokku@198.211.127.154:blogtest'
```

This issue is described in https://github.com/dokku/dokku/issues/2034 and should be fixed in https://github.com/dokku/dokku/pull/2205 according to Dokku. But I think it's not released in the Dokku version that we install yet.

**Comment by jvanbaarsen** [comment]
@michiels We are a bit behind on Docker versions. I'll create an issue to update to latest (They release like crazy :D )

**Comment by michiels** [comment]
@jvanbaarsen ok great! thanks for clearing up

**Comment by jvanbaarsen** [comment]
This should be fixed by: #160 (You might need to update your server)
